src/test/java/Homework16.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public static void ClickOnRegistrationLinkTest() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example public static void **clickOnRegistrationLinkTest()**

Reply:
Hello 
Your homework 16 is checked
You’re doing great! Keep it up! Very fine! 
💖Best wishes, the team HW - Alex!💖
E-mail: [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io)

![image](https://user-images.githubusercontent.com/104761482/228898831-86973a47-5563-46cf-b26a-22351ac82d96.png)